src/main.rs

```diff
@@ -175,42 +175,39 @@ async fn on_error(error: poise::FrameworkError<'_, Data, Error>) {
 
 #[tokio::main]
 async fn main() {
-    let matches = App::new("TTCBot")
+    let matches = Command::new("TTCBot")
```

Comment:
`clap` args should probably be transitioned to be using the derive API, to make working with them easier and generally nicer.